streampark-console/streampark-console-webapp/src/enums/flinkEnum.ts

```diff
@@ -140,11 +140,17 @@ export enum ClusterStateEnum {
   /** The cluster was just created but not started */
   CREATED = 0,
   /** cluster started */
-  STARTED = 1,
+  RUNNING = 1,
   /** cluster canceled */
   CANCELED = 2,
   /** cluster lost */
   LOST = 3,
+  /** cluster unknown */
+  UNKNOWN = 4,
+  STARTING = 5,
```

Comment:
These values are too arbitrary.  We can set the status according to certain rules, such as transitioning from "create" to "start" to "stop." and then list some status of each stage, which would be better